render/plugins/Genius.ts

```diff
@@ -20,8 +20,15 @@ export class Genius extends SearchLyrics {
                     this.debug('Error fetching search', err, res);
                     return cb('Error response searching genius');
                 }
-                let songUrl = json.response.sections[0].hits[0].result.url;
-                this.getSong(songUrl, cb);
+                let i = 0;
```

Comment:
please use a for loop here. And a variable for the `5` number.

Reply:
Done